package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/helix-onedrive-support",
-  "version": "1.2.1",
+  "version": "1.3.1",
```

Comment:
why did you change this? ideally, if the deps don't change, don't change the package-lock.

Reply:
This is the version of helix-onedrive-support itself, and it didn’t match the one in package.json - shouldn’t they?